Build/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I think we can remove this file for now. I don't know how to generate universal usable "launchSettings"